README.md

```diff
@@ -1 +1,44 @@
-Authenticator Sample App
+# Authenticator sample app
+
+This sample application demonstrates an authenticator-only application that uses PingOne for Customers mobile SDK. An iOS developer can easily build a branded and customized Authenticator application using this sample.
+
+## What’s in the sample app?
+
+  - Full native application written in Swift 5.1, compatible for all iOS devices from iOS 10 or above.
+  - Integration with PingOne SDK version 1.2.0.
```

Comment:
`Integration with PingOne SDK version 1.2.0.` --> `Integration with PingOne for Customers mobile SDK.`
also this should be a link to the sdk (on **master** branch).

Reply:
Fixed